taipy/gui/gui.py

```diff
@@ -143,6 +143,14 @@ class Gui:
             The signature of the *on_init* callback function must be:
 
             - *state*: the `State^` instance of the caller.
+        on_page_load (Callable): The function that is called when a page is loaded.<br/>
```

Comment:
I'm going to cry for more information.
"is loaded" is slightly vague, as we mentioned .